docs/libcurl/symbols-in-versions

```diff
@@ -709,6 +709,8 @@ CURL_FORMADD_NULL               7.9.8
 CURL_FORMADD_OK                 7.9.8
 CURL_FORMADD_OPTION_TWICE       7.9.8
 CURL_FORMADD_UNKNOWN_OPTION     7.9.8
+CURL_GET_SSLVERSION             7.53.0
+CURL_GET_SSLVERSION_MAX         7.53.0
```

Comment:
If they're not supposed to be used externally, then they shouldn't be in the public header, right?

If they are in the public header, they should be in `symbols-in-versions` I think.